plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -3673,7 +3673,9 @@ function wc_logout_url( $redirect = '' ) {
  * @since 3.1.0
  */
 function wc_empty_cart_message() {
-	echo '<p class="cart-empty woocommerce-info">' . wp_kses_post( apply_filters( 'wc_empty_cart_message', __( 'Your cart is currently empty.', 'woocommerce' ) ) ) . '</p>';
+	echo '<div class="cart-empty">';
+	wc_print_notice( wp_kses_post( apply_filters( 'wc_empty_cart_message', __( 'Your cart is currently empty.', 'woocommerce' ) ) ), 'notice' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
+	echo '</div>';
```

Comment:
It looks like this change is causing some e2e tests to fail, because they assume the message container is `p.cart-empty`. Can probably just update the three tests: [1](https://github.com/woocommerce/woocommerce/blob/c0efe1f2f537720d6b2fa8db15f33b055bf5856a/plugins/woocommerce/tests/e2e-pw/tests/shopper/single-product.spec.js#L118), [2](https://github.com/woocommerce/woocommerce/blob/c0efe1f2f537720d6b2fa8db15f33b055bf5856a/plugins/woocommerce/tests/e2e-pw/tests/shopper/single-product.spec.js#L208), [3](https://github.com/woocommerce/woocommerce/blob/c0efe1f2f537720d6b2fa8db15f33b055bf5856a/plugins/woocommerce/tests/e2e-pw/tests/shopper/single-product.spec.js#L320)